### PR TITLE
master-qa-199845

### PR DESCRIPTION
**Pull request by Songyuewen**
https://liferay.atlassian.net/browse/LPS-199845

### CODE REVIEW

**Comment by Songyuewen** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/publications/PublicationsDM.testcase [hunk 1]
```diff
@@ -213,6 +213,47 @@ definition {
...
+
+		PublicationsNavigator.gotoPublicationEllipsisItem(
+			menuItem = "Publish",
+			publicationName = "Publication Name");
+
+		AssertElementPresent(
+			key_changeCategory = "Dynamic Data Mapping Structure Layout",
+			key_changeTitle = "Dynamic Data Mapping Structure Layout",
+			locator1 = "PublicationsChanges#DISCARD_CHANGE_BUTTON");
+	}
```
Comment (suilin) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/publications/PublicationsDM.testcase:258]: Can you add a step to assert there is no “Edit“ option displayed next to the DDM Structure?

Comment (Songyuewen) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/publications/PublicationsDM.testcase:258]: > Can you add a step to assert there is no “Edit“ option displayed next to the DDM Structure?

Add step assert `PUBLICATIONS_CHANGE_DROPDOWN` element is not present

**Comment by Songyuewen** [comment]
ci:test:sf

**Comment by suilin** [comment]
ci:forward

**Pull request closed without merging**